src/cmd/build.js

```diff
@@ -40,6 +40,8 @@ export type PackageCreatorParams = {|
   sourceDir: string,
   fileFilter: FileFilter,
   artifactsDir: string,
+  eventToPromise: typeof defaultEventToPromise,
```

Comment:
This is an injected dependency and by convention we are always separating the regular parameter (with the options that the function is supposed to get usually) from the injected parameters (which are usually redefined only by the unit tests).

And so, `eventToPromise` should be removed from the `PackageCreatorParams` and a new `PackageCreatorOptions` type should be created:

```js
export type PackageCreatorOptions = {|
  eventToPromise: typeof defaultEventToPromise,
|};
```